contentcuration/contentcuration/management/commands/exportchannel.py

```diff
@@ -252,40 +248,49 @@ def get_or_create_language(language):
     )
 
 
-def create_content_thumbnail(thumbnail_string, file_format_id=file_formats.PNG, preset_id=None, uploaded_by=None):
-    thumbnail_data = ast.literal_eval(thumbnail_string)
-    if thumbnail_data.get('base64'):
-        with tempfile.NamedTemporaryFile(suffix=".{}".format(file_format_id), delete=False) as tempf:
-            tempf.close()
-            write_base64_to_file(thumbnail_data['base64'], tempf.name)
-            with open(tempf.name, 'rb') as tf:
-                return create_file_from_contents(tf.read(), ext=file_format_id, preset_id=preset_id, uploaded_by=uploaded_by)
+def create_associated_thumbnail(ccnode, ccfilemodel):
```

Comment:
Separated out thumbnail method to either take the encoding or encode/resize the thumbnail if no encoding is available